### ISSUE TITLE
When creating a new template via the backend, non-existing twig-templates also appear

### ISSUE THREAD

**Issue by contaoacademy** [contao/contao#7632]
### Affected version(s)

5.3.15

### Description

When I create a template via the backend, Contao also offers me a .`html.twig` template under certain circumstances. If I select this template, I get an error message.
RuntimeException:
`Invalid template @Contao/christiane/news_full.html.twig`

**Example**:
I have created a `news_latest.html5` in my theme-template subfolder. If I create another Template the news_latest appears as `.html.twig` and  `.html5`

However, as there is currently no template for the `news_latest.html.twig`.

The problem only occurs if I have created the html5-template in a theme folder before. If there is a html5-Version directly in the root template folder, this error does not happen.

**Steps to reproduce:**
1. configure a theme folder for your templates
2. create a new `news_full.html5` template via the backend within the theme folder
3. create a new template via the backend and search for `news_full`
4. the `html5` and the `html.twig` now appear in the selection

![screen-20241014_wAFoV7Pm](https://github.com/user-attachments/assets/8f85d2cc-5535-40e3-a55b-b091828a5ab6)

```
RuntimeException:
Invalid template @Contao/demoacademy/news_latest.html.twig

  at vendor/contao/core-bundle/contao/dca/tl_templates.php:297
  at tl_templates::{closure}('@Contao/demoacademy/news_latest.html.twig', 'templates')
     (vendor/contao/core-bundle/contao/dca/tl_templates.php:373)
  at tl_templates->addNewTemplate(object(DC_Folder))
     (vendor/contao/core-bundle/contao/classes/Backend.php:321)
  at Contao\Backend->getBackendModule('tpl_editor', null)
     (vendor/contao/core-bundle/contao/controllers/BackendMain.php:144)
  at Contao\BackendMain->run()
     (vendor/contao/core-bundle/src/Controller/BackendController.php:44)
  at Contao\CoreBundle\Controller\BackendController->mainAction()
     (vendor/symfony/http-kernel/HttpKernel.php:181)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
     (vendor/symfony/http-kernel/HttpKernel.php:76)
  at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
     (vendor/symfony/http-kernel/Kernel.php:197)
  at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
     (public/index.php:42)
```

**Comment by fritzmg** [comment]
I can confirm the issue. One thing to note: you _must_ create the `<theme>/news_full.html5` template via the template editor first as well (I could not reproduce it by creating the initial template manually).

**Comment by aschempp** [comment]
@m-vo is this fixed in 5.5 with the new template studio? I would be fine with marking this as a known limitation of 5.3 and not invest time to fix this if it becomes obsolete anyway.

**Comment by m-vo** [comment]
Yes, the template studio only allows creating user templates for already existing app/bundle templates.
